test/junit/scala/tools/nsc/DeterminismTest.scala

```diff
@@ -187,6 +190,48 @@ class DeterminismTest {
     test(List(code))
   }
 
+  @Test def testAnnotations1(): Unit = {
+    def code = List[SourceFile](
+      source("a.scala",
+        """
+          |class Annot1(s: String) extends scala.annotation.StaticAnnotation
```

Comment:
Should it be annotated with `java.lang.annotation.Repeatable`? if it's not annotated with `@Repeatable` then  
 annotate the annotation multiple times will be an error right?

Reply:
I'll add a second test for `@Repeatable` annotations. This test was showing that the compiler change to support `@Repeatable` annotations changed the behaviour of regular, non-repeated annotations.